File: app/src/main/java/teamenglify/englify/Model/ReadPart.java
package teamenglify.englify.Model;

import java.io.Serializable;
import java.util.Date;

/**
 * Created by Keith on 07-Mar-17.
 */

public class ReadPart implements Serializable {
    public String reading;
    public String audioURL;
    public String imgURL;
    public Date lastModified;

    public ReadPart(String reading, String imgURL, String audioURL) {
        this.reading = reading;
        this.audioURL = audioURL;
        this.imgURL = imgURL;
    }

    public ReadPart(String reading, String imgURL, String audioURL, Date lastModified) {
        this.reading = reading;
        this.audioURL = audioURL;
        this.imgURL = imgURL;
        this.lastModified = lastModified;
    }
}

File: app/src/main/java/teamenglify/englify/Model/realm/ReadPart.kt
package teamenglify.englify.Model.realm

import io.realm.RealmObject

open class ReadPart: RealmObject() {
    //From firebase realtime store
    lateinit var id: String
    lateinit var reading: String
    var translation: String? = null
    var imageUrl: String? = null
    var audioUrl: String? = null

    //From firebase storage
    var imageData: ByteArray? = null
    var audioData: ByteArray? = null
}